src/KiwiLogin/index.js

```diff
@@ -123,7 +123,7 @@ type State = {|
   showError: boolean,
 |};
 
-class KiwiLogin extends React.Component<Props, State> {
+export class PureKiwiLogin extends React.Component<Props, State> {
```

Comment:
We can discuss about naming of the pure components which are exported without HOC stuff

Reply:
I saw you call it `RawComponent,` so I will follow that naming convention ;)